### PR TITLE
Added RxJava2 to java-vertx server codegen

### PR DESCRIPTION
**Pull request by jharmelink**
RxJava2 uses another base package. To support this package in java-vertx server codegen, a `rxVersion2` `boolean` flag/option was added which defaults to `false`.

@phiz71 @wing328

### CODE REVIEW

File: modules/openapi-generator-maven-plugin/src/main/java/org/openapitools/codegen/plugin/CodeGenMojo.java [hunk 1]
```diff
@@ -245,7 +245,7 @@
      * To treat a document strictly against the spec.
      */
     @Parameter(name = "strictSpec", property = "openapi.generator.maven.plugin.strictSpec", required = false)
-    private Boolean strictSpecBehavior;
+    private Boolean strictSpec;
```
Comment (wing328) [inline modules/openapi-generator-maven-plugin/src/main/java/org/openapitools/codegen/plugin/CodeGenMojo.java]: @jharmelink are these changes in this file intentional?

**Comment by wing328** [comment]
@jharmelink from what I understand, this is not a breaking change so you can file the change against the master branch (next patch release will be in a week)

File: modules/openapi-generator-maven-plugin/src/main/java/org/openapitools/codegen/plugin/CodeGenMojo.java (hunk 1 above)
Comment (jharmelink) [reply modules/openapi-generator-maven-plugin/src/main/java/org/openapitools/codegen/plugin/CodeGenMojo.java]: This is actually a bugfix to get the generation working again. Not related to this change.

**Comment by jharmelink** [comment]
I'll create a new PR for master

File: modules/openapi-generator-maven-plugin/src/main/java/org/openapitools/codegen/plugin/CodeGenMojo.java (hunk 1 above)
Comment (wing328) [reply modules/openapi-generator-maven-plugin/src/main/java/org/openapitools/codegen/plugin/CodeGenMojo.java]: The issue should have been fixed in the master. Please have a look and let us know if that's not the case

**Comment by jharmelink** [comment]
Created new PR for master: https://github.com/OpenAPITools/openapi-generator/pull/3240

**Pull request closed without merging**